lib/src/showcase.dart

```diff
@@ -250,56 +262,75 @@ class _ShowcaseState extends State<Showcase> with TickerProviderStateMixin {
     Size size,
     Rect rectBound,
     Size screenSize,
-  ) =>
-      Visibility(
-        visible: _showShowCase,
-        maintainAnimation: true,
-        maintainState: true,
-        child: Stack(
-          children: [
-            GestureDetector(
-              onTap: _nextIfAny,
-              child: Container(
-                width: MediaQuery.of(context).size.width,
-                height: MediaQuery.of(context).size.height,
-                child: CustomPaint(
-                  painter: ShapePainter(
-                    opacity: widget.overlayOpacity,
-                    rect: position!.getRect(),
-                    shapeBorder: widget.shapeBorder,
+  ) {
+    var blur = widget.blur == null
+        ? ShowCaseWidget.of(context)!.defaultBlur
```

Comment:
please remove force unwrap

Reply:
I do not understand. can you please provide more desc.

Reply:
ShowCaseWidget.of(context)! is force unwrap if there will not be any ShowCaseWidget.of(context) then it will throw an exception so instead of force unwrap we can use null aware operator. ShowCaseWidget.of(context)?. defaultBlur ?? default value.